lib/assessment.js

```diff
@@ -32,14 +42,14 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
-
+  return array.filter(callback);
 };
 
 // Question 5
 // Write a function that returns the first element
 // for which invoking the callback returns a truthy value
 var find = function find(array, callback, thisArg) {
-
+  return array.filter(callback)[0];
```

Comment:
Same as above, except this shows that you know the relationship between `find` and `select`. Doing it this way is wasteful, though, since you build the whole list just to extract its zeroth element. 1/4 credit.